packages/synapse-react-client/src/mocks/user/mock_user_profile.ts

```diff
@@ -25,9 +20,9 @@ export const mockUserProfileData = {
   createdOn: '2018-06-18T21:42:48.000Z',
   url: 'https://sagebase.org',
   email: 'mockemail@fakeemailprovider.xyz',
-} satisfies UserProfile
+} satisfies Readonly<UserProfile>
```

Comment:
Starting to add `Readonly` to mock data, because mutating mock objects shared between tests has been a source of bugs in the past. I haven't attempted to do this systemically